.github/workflows/gitpod.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   build:
     name: Build Gitpod Docker image 
     runs-on: ubuntu-latest
-    environment: scipy-dev
```

Comment:
This refers to the GitHub Scipy environment: https://github.com/scipy/scipy/deployments/activity_log?environment=scipy-dev

So this should not be changed as this has the DockerHub token and username